### ISSUE TITLE
502 issue on production

### ISSUE THREAD

**Issue by MatteoGauthier** [pastelsky/tsdocs#48]
Hey ! I just got a 502 error on the tsdocs.dev website

<img width="582" alt="Capture d’écran 2024-02-07 à 00 51 21" src="https://github.com/pastelsky/tsdocs/assets/32040951/9450a6c5-80ce-4637-b38c-9005ae75cfb2">

**Comment by shiftgeist** [comment]
Content is still accessible at `https://tsdocs.dev/search/docs/{PACKAGE_NAME}`

**Comment by impressivewebs** [comment]
> Content is still accessible at `https://tsdocs.dev/search/docs/{PACKAGE_NAME}`

Doesn't seem to work. For example: https://tsdocs.dev/search/docs/@angular/core

I either get a 502 error or it just hangs the browser tab.

**Comment by kravetsone** [comment]
@pastelsky We need your help!

**Comment by shiftgeist** [comment]
> > Content is still accessible at `https://tsdocs.dev/search/docs/{PACKAGE_NAME}`
> 
> Doesn't seem to work. For example: https://tsdocs.dev/search/docs/@angular/core
> 
> I either get a 502 error or it just hangs the browser tab.

we might be back in business, that link resolves for me

**Comment by impressivewebs** [comment]
Yes, seems to be working now. Thanks for the heads-up!

**Comment by gdude2002** [comment]
Still getting 502s here - for example, https://tsdocs.dev/docs/electron-util

**Comment by nmanumr** [comment]
I'm still getting the same 502

**Comment by tiagosiebler** [comment]
Seems to be having problems again? Bad gateway

**Comment by tiagosiebler** [comment]
@pastelsky Hi, any plans to bring this back? If not, would you mind sharing a repo for what was running on tsdocs.dev, so I can run my own? Thanks!

**Comment by FleetAdmiralJakob** [comment]
Yeah, would be cool for a "deploy it yourself" guide because I don't want to set it up locally every time, that would be too annoying.

**Comment by pastelsky** [comment]
The service has been provisionally restored. 
At the moment, the service is on a barebones droplet so don't expect it to be particularly fast or completely reliable, but hopefully that improves if I can find better funding for this project.

**Comment by tiagosiebler** [comment]
> The service has been provisionally restored. At the moment, the service is on a barebones droplet so don't expect it to be particularly fast or completely reliable, but hopefully that improves if I can find better funding for this project.

Thank you! If you could provide a repo with everything needed to build an image of the deployed service, I could look at hosting a mirror on one (or two, depending on demand) of my servers. Any version I've tried to run doesn't have the same look & feel as your deployed version.
